### PR TITLE
BAEL-7026: Check If a String Contains a Number Value in Java

### PR DESCRIPTION
**Pull request by azhwani**
+ Create new methods for checking if a string contains a digit
+ Create unit test cases for the methods

### CODE REVIEW

File: core-java-modules/core-java-string-operations-6/src/main/java/com/baeldung/strcontainsnumber/StrContainsNumberUtils.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    static boolean checkUsingGuava(String input) {
+        if (input == null || input.isEmpty()) {
+            return false;
+        }
+
+        String result = CharMatcher.forPredicate(Character::isDigit)
+          .retainFrom(input);
+
+        return result.length() > 0;
```
Comment (JavierPeris) [inline core-java-modules/core-java-string-operations-6/src/main/java/com/baeldung/strcontainsnumber/StrContainsNumberUtils.java:74]: We can replace `result.length() > 0` with `!result.isEmpty()`

File: core-java-modules/core-java-string-operations-6/src/main/java/com/baeldung/strcontainsnumber/StrContainsNumberUtils.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            return false;
+        }
+
+        return input.chars()
+          .anyMatch(Character::isDigit);
+    }
+
+    static boolean checkUsingApacheCommonsLang(String input) {
+        String result = StringUtils.getDigits(input);
+        return result != null && result.length() > 0;
```
Comment (JavierPeris) [inline core-java-modules/core-java-string-operations-6/src/main/java/com/baeldung/strcontainsnumber/StrContainsNumberUtils.java:63]: We forgot to replace this length check.
```suggestion
        return result != null && !result.isEmpty();
```

Comment (azhwani) [reply core-java-modules/core-java-string-operations-6/src/main/java/com/baeldung/strcontainsnumber/StrContainsNumberUtils.java:63]: Good catch ;) done

File: core-java-modules/core-java-string-operations-6/src/main/java/com/baeldung/strcontainsnumber/StrContainsNumberUtils.java (hunk 1 above)
Comment (azhwani) [reply core-java-modules/core-java-string-operations-6/src/main/java/com/baeldung/strcontainsnumber/StrContainsNumberUtils.java]: done

**Review by JavierPeris — APPROVED** [review]

**Pull request merged**